Avalonia.MusicStore/Roots.xml

```diff
@@ -1,5 +1,4 @@
 ﻿<linker>
```

Comment:
This file should be removed, if we use compiled bindings everywhere. Just need to double check with AOT

Reply:
ViewLocator ;-)